projects/onto-search/src/lib/onto-search-facet/range-facet/range-histogram/range-histogram.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="canvas">
```

Comment:
Do you think this will be the only canvas in the app? Should we have more specific id?

Reply:
renamed